library/core/src/macros/mod.rs

```diff
@@ -713,7 +713,7 @@ macro_rules! unimplemented {
 /// Indicates unfinished code.
 ///
 /// This can be useful if you are prototyping and are just looking to have your
-/// code typecheck.
+/// code typechecked.
```

Comment:
I don't think that's what they meant. By "have your code typecheck" they probably meant "have your code pass typecheck so that it compiles successfully"